src/components/AddTask.js

```diff
@@ -0,0 +1,55 @@
+import { useState } from 'react';
+import { BsPlus, BsFillTrashFill } from 'react-icons/bs';
+import { FiEdit } from 'react-icons/fi';
+import Modal from './Modal';
+function AddTask() {
+  const [showModal, setShowModal] = useState(false);
+  const handleShowModal = toggle => {
+    setShowModal(toggle);
```

Comment:
Could have directly done !showModal instead of passing toggle everytime

Reply:
Yaa.. could be shown..
Looks like unnecessary state managements